### DISCUSSION TITLE
Option to convert to a currency only the Summary line

### DISCUSSION THREAD

**Discussion by gabrielpres** [achannarasappa/ticker#130] [Ideas]
Hi, again congrats on this very useful app. If I may add a feature suggestion, it would be the option of showing only the summary (total value, total change, total daily change) in a different currency, while leaving the currencies of the lots and watchlist in the relevant market currencies. I think this would leave the correct level of detail for the single lots, while giving the useful info of total value in the user's local currency. If I understand correctly, this is not the current behaviour, as all prices and cost (lots, watchlist) would be changed to the selected currency. Thanks.

**Comment by viperperidot** [comment]
I could see this option being useful in addition to having the current config option as well.
